src/test/java/LoginTests.java

```diff
@@ -1,21 +1,52 @@
-import org.openqa.selenium.WebDriver;
-import org.openqa.selenium.chrome.ChromeDriver;
+import Pages.HomePage;
+import Pages.LoginPage;
+import io.cucumber.java.After;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.WebDriverWait;
 import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Parameters;
 import org.testng.annotations.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
 
+import java.net.MalformedURLException;
 import java.time.Duration;
 
 public class LoginTests extends BaseTest {
+    
+    @BeforeMethod
+    @Parameters({"BaseUrl"})
+    public void launchBrowser_main(String BaseUrl) throws MalformedURLException {
+        this.launchBrowser(BaseUrl);
+    }
+
+    @AfterMethod
+    public void closeBrowser_main() {
+        this.closeBrowser();
+    }
```

Comment:
Please, remove code redundancies

Don't repeat yourself! ( In Java, it means don't write the same code repeatedly. )